pkg/apis/serving/v1alpha1/kfservice_status.go

```diff
@@ -88,9 +88,10 @@ func (ss *KFServiceStatus) PropagateCanaryConfigurationStatus(canaryConfiguratio
 }
 
 // PropagateRouteStatus propagates route's status to the service's status.
-func (ss *KFServiceStatus) PropagateRouteStatus(rs *knservingv1alpha1.RouteStatus) {
-	ss.URL = rs.Domain
-
+func (ss *KFServiceStatus) PropagateRouteStatus(rs *knservingv1beta1.RouteStatus) {
+	if rs.URL != nil {
+		ss.URL = rs.URL.Host
```

Comment:
Can we ducktype our status to be like theirs? I expect we also want to have scheme, etc.

Reply:
So, you mean changing our type to URL too?